### ISSUE TITLE
Header and Footer are not getting printer during conversion of a banner file using bannertopdf.

### ISSUE THREAD

**Issue by sahilarora535** [OpenPrinting/cups-filters#15]
The `Header` and `Footer` keywords in a banner file are not getting printed when converted using `bannertopdf` filter. Here are the contents of the banner file:
```
#PDF-BANNER
Template default.pdf
Header Hello
Footer World
Show job-id
```
The file `default.pdf` can be found [here](https://github.com/OpenPrinting/cups-filters/blob/master/data/default.pdf).

The resulting PDF file do have `job-id`, or other information which is specified in the `Show` keyword, but no header or footer.

Here is the log for the conversion:
```
DEBUG: argv[0]="cupsfilter"
DEBUG: argv[1]="1"
DEBUG: argv[2]="sahil"
DEBUG: argv[3]="default"
DEBUG: argv[4]="1"
DEBUG: argv[5]=""
DEBUG: argv[6]="default"
DEBUG: envp[0]="<CFProcessPath>"
DEBUG: envp[1]="CONTENT_TYPE=application/vnd.cups-pdf-banner"
DEBUG: envp[2]="CUPS_DATADIR=/usr/share/cups"
DEBUG: envp[3]="CUPS_FONTPATH=/usr/share/cups/fonts"
DEBUG: envp[4]="CUPS_SERVERBIN=/usr/lib/cups"
DEBUG: envp[5]="CUPS_SERVERROOT=/etc/cups"
DEBUG: envp[6]="LANG=en_US.UTF8"
DEBUG: envp[7]="PATH=/usr/lib/cups/filter:/usr/bin:/usr/sbin:/bin:/usr/bin"
DEBUG: envp[8]="PPD=/usr/share/cups/model/laserjet.ppd"
DEBUG: envp[9]="PRINTER_INFO=cupsfilter"
DEBUG: envp[10]="PRINTER_LOCATION=Unknown"
DEBUG: envp[11]="PRINTER=cupsfilter"
DEBUG: envp[12]="RIP_MAX_CACHE=128m"
DEBUG: envp[13]="USER=sahil"
DEBUG: envp[14]="CHARSET=utf-8"
DEBUG: envp[15]="FINAL_CONTENT_TYPE=application/pdf"
INFO: bannertopdf (PID 22483) started.
DEBUG: Could not open PPD file '/usr/share/cups/model/laserjet.ppd'
PDF template file doesn't have form. It's okay.
INFO: bannertopdf (PID 22483) exited with no errors.
```

**Comment by tillkamppeter** [comment]
Same as with issue #14.

**Comment by tillkamppeter** [comment]
Should be solved by the solution of issue #8.
